src/main/java/com/higgsup/kpi/service/impl/GroupServiceImpl.java

```diff
@@ -231,7 +230,7 @@ public GroupDTO updateSupport(GroupDTO<GroupSupportDetail> groupDTO) throws Json
         Optional<KpiGroupType> kpiGroupType = kpiGroupTypeRepo.findById(groupDTO.getGroupTypeId().getId());
         if (kpiGroupRepo.findByGroupTypeId(kpiGroupType) == null) {
             groupDTO.setErrorCode(ErrorCode.NOT_FIND.getValue());
-            groupDTO.setMessage(ErrorMessage.NOT_FIND_GROUP_TYPE);
+            groupDTO.setMessage(ErrorMessage.NOT_FIND_GROUP);
```

Comment:
vì sao lại chuyển từ  mes " group type does not exist " sang  " not find group " ??

Reply:
đang sai nội dung, chỗ đấy message phải là k tìm thấy group chứ k phải k tìm thấy group type

Reply:
ukm, em có nhìn qua trong code của anh, ở phần này thì anh sửa lại thế này là oki, nhưng có 1 vấn đề đó là nếu người dùng gửi type id sai, thì trương trình sẽ chạy bình thường sau đó đến code dòng 216 sẽ có lỗi, và lỗi này sẽ không báo lên client , code đoạn này thiếu validate KpiGroupType

Reply:
đoạn code này sẽ có 1 bug khi người dùng gửi id type sai.